.github/workflows/ci.yml

```diff
@@ -54,3 +54,14 @@ jobs:
           javac -J-Xmx32m -version
           mvn install -DskipTests=true -Dmaven.javadoc.skip=true -B -V
           mvn test -B
+  test-js:
+    name: Test JS
+    needs: build
+    if: |
+      !cancelled() && !failure()
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/setup-node@v4
+        with:
+          node-version: 20
+      - run: node --
```

Comment:
```suggestion
      - run: node --test
```

Reply:
![image](https://github.com/shapesecurity/salvation/assets/6004480/4d8ad7cb-9052-4377-af91-c2276821d7ba)

How did I do that